src/server/app.ts

```diff
@@ -40,7 +40,7 @@ const isDev = app.get("env") === "development";
 const isTrustedProxy = SERVER_SETTINGS.getTrustProxy() === "always";
 
 if (isTrustedProxy) {
-  app.set("trust proxy", 1); // trust first proxy
+  app.set("trust proxy", true); // trust first proxy
```

Comment:
Could you please update this comment? Express will no longer trust only the first proxy.